### PR TITLE
IGVF-613-staging-documentation

### PR DESCRIPTION
**Pull request by gabdank**

### CODE REVIEW

File: docs/staging_deploy.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+5. After approval of the PR to `main` follow next steps
+
+6. Merge the PR to `main`.
+```
+$ git checkout dev
+$ git fetch origin -p
+$ git pull
+$ git checkout main
+$ git pull
+$ git merge dev --ff-only
```
Comment (keenangraham) [inline docs/staging_deploy.md:19]: Only proceed if there are no merge errors here.

File: docs/rebuild_stacks_with_prod_DB.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+![synth output](images/staging_aws.png)
+
+2. Go to `CloudFormation/Stacks` and search for the relevant stacks, for example search for `staging`
+![synth output](images/staging_stacks.png)
+
+3. Select `BackendStack` and `Delete` it
+
+4. Only after `BackendStack` has finished to be deleted proceed to `PostgresStack` and `OpensearchStack`
+
+5. If you merge the `PR` to main after these actions, the stacks you deleted would be rebuild with the new DB snapshot.
```
Comment (keenangraham) [inline docs/rebuild_stacks_with_prod_DB.md:13]: `stacks you deleted would be rebuild with the new DB snapshot.` -> `the deleted stacks will be recreated with the new DB snapshot`?

File: docs/rebuild_stacks_with_prod_DB.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+1. Log in to igvf-staging AWS account
+![synth output](images/staging_aws.png)
+
+2. Go to `CloudFormation/Stacks` and search for the relevant stacks, for example search for `staging`
+![synth output](images/staging_stacks.png)
+
+3. Select `BackendStack` and `Delete` it
+
+4. Only after `BackendStack` has finished to be deleted proceed to `PostgresStack` and `OpensearchStack`
```
Comment (keenangraham) [inline docs/rebuild_stacks_with_prod_DB.md:11]: `has finished to be deleted proceed` -> `has finished deleting`?

**Review by keenangraham — CHANGES_REQUESTED** [review]

File: docs/staging_deploy.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+```
+$ git checkout main
+$ git fetch origin -p
+$ git pull
+$ git log --pretty
+```
+
+9. After choosing the commits to be included in the tag. Make the tag for version `X.Y.0`:
+```
+$ export igvfd_version=X.Y.0
```
[the remark this answers is not in the document]
Comment (gabdank) [reply docs/staging_deploy.md:35]: updated

File: docs/staging_deploy.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+8. Copy the list of commits that have been merged to `main`:
+```
+$ git checkout main
+$ git fetch origin -p
+$ git pull
+$ git log --pretty
+```
+
+9. After choosing the commits to be included in the tag. Make the tag for version `X.Y.0`:
```
[the remark this answers is not in the document]
Comment (gabdank) [reply docs/staging_deploy.md:33]: added checkout dev before

File: docs/staging_deploy.md (hunk 1 above)
Comment (gabdank) [reply docs/staging_deploy.md:19]: added

File: docs/rebuild_stacks_with_prod_DB.md (hunk 3 above)
Comment (gabdank) [reply docs/rebuild_stacks_with_prod_DB.md:11]: updated

**Review by keenangraham — APPROVED** [review]

**Pull request merged**